tile/platform/local_machine/program.cc

```diff
@@ -170,6 +170,12 @@ lang::KernelList CompileProgram(const lang::RunInfo& runinfo,  //
 
 }  // namespace
 
+// Static members of Program
+std::once_flag Program::first_run_;
```

Comment:
It's just because they are related with Program and only Program uses them. Otherwise, others can use them. Of course, it doesn't matter. I'll change it if you insist.

Reply:
If they're in an unnamed namespace in `program.cc`, nothing else can use them.  :-)
I do think it'd be more clear that they're global, not per-instance.

Reply:
Personally I think it's fine to have statics on Program if they are related to global Program state.

Reply:
The only real problem I have with them is that it's not immediately obvious when reading the code whether a variable is instance-local or instance-global.  Making them non-members alters the names and makes it obvious, and since they're not accessed outside of `program.cc`, there's no hard.  IMHO, static class members are useful when multiple translation units need to know about the same members; that's not the case here.